### PR TITLE
fix: switch size

### PR DESCRIPTION
**Pull request by eljefe223**
# Description
This fixes `fast-switch` size and disabled state. Disabled was also broken in `fast-checkbox` so fixing it here. Also there was an extra closing tag in `fast-badge` base.html so I removed it.

After:
image

## Issue type checklist

- [x] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist

- [x] I have added tests for my changes.
- [ ] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: packages/fast-components/src/checkbox/checkbox.ts [hunk 1]
```diff
@@ -38,6 +38,9 @@ export class Checkbox extends FormAssociated<HTMLInputElement> {
         this.defaultChecked = this.checkedAttribute;
     }
 
+    @attr
+    public disabled: boolean;
+
```
Comment (chrisdholt) [inline packages/fast-components/src/checkbox/checkbox.ts:43]: @eljefe223 this seems unrelated; can we remove?

Comment (chrisdholt) [reply packages/fast-components/src/checkbox/checkbox.ts:43]: Also, this is inheriting from form associated - if you were seeing an issue with disabled it was related to #2893 which has since been fixed.

File: packages/fast-components/src/checkbox/checkbox.ts [hunk 2]
```diff
@@ -38,6 +38,9 @@ export class Checkbox extends FormAssociated<HTMLInputElement> {
         this.defaultChecked = this.checkedAttribute;
     }
 
+    @attr
+    public disabled: boolean;
```
Comment (EisenbergEffect) [inline packages/fast-components/src/checkbox/checkbox.ts:42]: Do we need to make this `@attr({ mode: 'boolean' })` or `@attr({mode: 'fromView', converter: booleanConverter })` Ping @chrisdholt for the latest recommendation based on new components.

File: packages/fast-components/src/switch/switch.ts [hunk 3]
```diff
@@ -33,6 +33,9 @@ export class Switch extends FormAssociated<HTMLInputElement> {
         this.defaultChecked = this.checkedAttribute;
     }
 
+    @attr
+    public disabled: boolean;
```
Comment (EisenbergEffect) [inline packages/fast-components/src/switch/switch.ts:37]: Same question as above.

File: packages/fast-components/src/checkbox/checkbox.ts (hunk 1 above)
Comment (eljefe223) [reply packages/fast-components/src/checkbox/checkbox.ts:43]: Ok, Didn't realize their was a bug tracking, adding this attribute fixed this but I can remove.

File: packages/fast-components/src/badge/fixtures/base.html [hunk 4]
```diff
@@ -54,7 +54,7 @@ <h4>Circular</h4>
     <fast-badge fill="secondary" color="foo" id="foo" circular="true">
         99
     </fast-badge>
-    <fast-badge fill="tertiary" color="bar" id="foo" circular="true"></fast-badge>
+    <fast-badge fill="tertiary" color="bar" id="foo" circular="true">
         9999999
     </fast-badge>
```
Comment (chrisdholt) [inline packages/fast-components/src/badge/fixtures/base.html:59]: I missed this one - but we can probably do in a different PR

File: packages/fast-components/src/switch/switch.styles.ts [hunk 5]
```diff
@@ -63,8 +64,8 @@ export const SwitchStyles = css`
 
     .checked-indicator {
         position: absolute;
-        height: calc((${heightNumber} - (var(--design-unit) * 2)) * 1px);
-        width: calc((${heightNumber} - (var(--design-unit) * 2)) * 1px);
+        height: calc((${heightNumber} - (var(--design-unit) * 5.5)) * 1px);
```
Comment (bheston) [inline packages/fast-components/src/switch/switch.styles.ts:67]: I'm not going to block on this since I'm behind in working on the spec for handling density, but I'd like to see us calculate sizes by something that provides some declarative way to understand what's going on and avoid magic number or fractions that mostly happen to produce the desired value. I'll look into this more holistically though.
